src/components/datepicker/js/datepickerDirective.js

```diff
@@ -646,6 +683,10 @@
       self.isCalendarOpen = false;
       self.ngModelCtrl.$setTouched();
 
+      if (self.$attrs.mdIsOpen) {
```

Comment:
You can probably make the `mdIsOpen` scope variable to optional and just check here for it being defined.

```js
{
  isOpen: '=?mdIsOpen' 
}

angular.isDefined(this.isOpen)
```